en/services/high_latency.md

```diff
@@ -43,7 +45,20 @@ Enum | Description
 
 ## Sequences
 
-A GCS will typically have one primary link that is used for all vehicle communications, but may also have secondary links that it can switch to if needed (which will then become the primary link).
+A GCS will typically have one or more (low latency) links that are used for vehicle communications. Separate to this are any high latency links. The high latency links may be active simultaneous to the low latency links during handover operations.
+
+A typical flight sequence will look like:
+- Vehicle starts up on the ground with low latency links active
+- Ground station(s) download, check and sync all [mission protocol](../services/mission.md) items and [parameter protocol](../services/parameter.md) items over low latency link
+- Vehicle starts mission
+- Whilst still in range of low latency link, enable high latency link and confirm transmission of messages over high latency link.
+- Vehicle moves out of range of low latency link.
+
+The reverse occurs at the end of a mission.
+
+The GCS and autopilot should be able to work with a mixed regime of low and high latency links. Any [command protocol](../services/command.md) messages should be sent over all available links.
+
+If used, the high latency link should be enabled (MAV_CMD_CONTROL_HIGH_LATENCY message) over the low latency link prior to loss of coverage. This ensures no break in telemetry. (i.e sending the MAV_CMD_CONTROL_HIGH_LATENCY on the high latency link _after_ loss of the low latency link may mean a wait of several 10's of seconds before the first HIGH_LATENCY2 message appears at the GCS).
```

Comment:
As above, this is not a requirement and depends on the use case. Consider a balloon - it might only ping you once every 3 hours (say). I'll reword.